packages/react-native/example/package.json

```diff
@@ -11,7 +11,7 @@
     "postinstall": "cd node_modules/@notifee/react-native && yarn"
   },
   "dependencies": {
-    "@notifee/react-native": "github:notifee/react-native-notifee",
+    "@notifee/react-native": "link:../",
```

Comment:
I linked to the current project to check the build with changes, please let me know if a revert is needed!

Reply:
I feel like this was there for some reason, but in retrospect I can't remember what it is. I think the link to local is better personally and I'd like it to work. Willing to go with it and see if it works, personally